src/python/bot/fuzzers/options.py

```diff
@@ -88,11 +90,12 @@ def __init__(self, options_file_path, cwd=None):
     else:
       self._cwd = os.path.dirname(options_file_path)
 
-    self._config = ConfigParser.ConfigParser()
-    try:
-      self._config.read(options_file_path)
-    except ConfigParser.Error:
-      raise FuzzerOptionsException('Failed to parse fuzzer options file.')
+    self._config = configparser.ConfigParser()
+    with open(options_file_path, 'r') as f:
```

Comment:
This was changed to an explicit file open to make pyFakeFS work, since python-future seems to patch something in a way such that it doesn't work with open() here anymore.